packages/example/src/components/Transactions/TransactionForm.tsx

```diff
@@ -0,0 +1,285 @@
+import { formatEther } from '@ethersproject/units'
+import { TransactionStatus, useEthers } from '@usedapp/core'
+import React, { ReactElement, useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { TextBold } from '../../typography/Text'
+import { ContentBlock } from '../base/base'
+import { Button } from '../base/Button'
+import { BorderRad, Colors } from '../../global/styles'
+import { BigNumber } from 'ethers'
+import { SpinnerIcon, CheckIcon, ExclamationIcon } from './Icons'
+
+import { AnimatePresence, motion } from 'framer-motion'
+
+const formatter = new Intl.NumberFormat('en-us', {
+  minimumFractionDigits: 4,
+  maximumFractionDigits: 4,
+})
+
+const formatBalance = (balance: BigNumber | undefined) =>
+  formatter.format(parseFloat(formatEther(balance ?? BigNumber.from('0'))))
+
+interface StatusBlockProps {
+  color: string
+  text: string
+  icon: ReactElement
+}
+
+const StatusBlock = ({ color, text, icon }: StatusBlockProps) => (
+  <InformationRow
+    layout
+    initial={{ opacity: 0, y: -50 }}
+    animate={{ opacity: 1, y: 0 }}
+    exit={{ opacity: 0, y: 50 }}
+    key={text}
+  >
+    <IconContainer style={{ fill: color }}>{icon}</IconContainer>
+    <div style={{ color: color, textAlign: 'center' }}>{text}</div>
+  </InformationRow>
+)
+
+const transactionErrored = (transaction: TransactionStatus) => {
+  return 'errorMessage' in transaction
+}
+
+const getTransactionErrorMessage = (transaction: TransactionStatus) => {
+  if ('errorMessage' in transaction) return transaction.errorMessage
```

Comment:
No need to extract helper, just use `transaction?.errorMessage`

Reply:
See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html

Reply:
It's correct this way, TranscationStatus without checks is just 
```js
{
      status: string
}
```

Reply:
Fair enough :)
I was wondering if we can do:
```ts
transactionErrored(transaction) && transaction?.errorMessage
```
Does not seem awesome.